domain/src/main/java/co/nimblehq/compose/crypto/domain/model/CoinItem.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.compose.crypto.domain.model
+
+data class CoinItem(
+    val id: String = "",
+    val symbol: String = "",
+    val coinName: String = "",
+    val currentPrice: Int = 0,
+    val marketCap: Long = 0,
+    val marketCapRank: Int = 0,
+    val totalVolume: Long = 0,
+    val high24h: Long = 0,
+    val low24h: Long = 0,
+    val priceChange24h: Double = 0.0,
+    val priceChangePercentage24h: Double = 0.0,
+    val marketCapChange24h: Double = 0.0,
+    val marketCapChangePercentage24h: Double = 0.0,
+    val priceChangePercentage24hInCurrency: Double = 0.0
```

Comment:
Regarding this [comment](https://github.com/nimblehq/jetpack-compose-crypto/pull/23/files#r956874420) I don't think we need to set the default values here 🤔 
```suggestion
    val id: String,
    val symbol: String,
    val coinName: String,
    val currentPrice: Int,
    val marketCap: Long,
    val marketCapRank: Int,
    val totalVolume: Long,
    val high24h: Long,
    val low24h: Long,
    val priceChange24h: Double,
    val priceChangePercentage24h: Double,
    val marketCapChange24h: Double,
    val marketCapChangePercentage24h: Double,
    val priceChangePercentage24hInCurrency: Double
```

Reply:
Ah, we might need to fallback to default value in extension when converting response to model instead @hoangnguyen92dn  🙏